pylint/reporters/text.py

```diff
@@ -37,6 +37,24 @@ class MessageStyle(NamedTuple):
     style: tuple[str, ...] = ()
     """Tuple of style strings (see `ANSI_COLORS` for available values)."""
 
+    def get_ansi_code(self) -> str:
```

Comment:
Let's make this private

Reply:
It's a little complicated to make a public class function private (then we need to call it with ``.__MessageStyle_`` ... I'm wondering if we could use ``colorama`` entirely instead. Or make this a public API, it's unlikely it's ever going away.

Reply:
I don't understand your comment. We can just make the function private and keep `MessageStyle` public right? We have similar patterns for all of our other classes: the class itself is public but not all method are public.

Reply:
```python
>>> class MessageStyle:
...     def __ansi_color(self):
...         print("ansi")
... 
>>> m = MessageStyle()
>>> m.__ansi_color()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'MessageStyle' object has no attribute '__ansi_color'
>>> m._MessageStyle__ansi_color()
ansi
```

Reply:
I meant `_get_ansi_code`. That signals that it is private and not part of `MessageStyle`'s API

Reply:
Well it could be made private with a small refactor [64e5b95](https://github.com/PyCQA/pylint/pull/8412/commits/64e5b95986604ae3265ad839b0b137bae1799669).